### PR TITLE
PLF-5064 Avoid using PID in prameter when it doesn't exist

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: plf-tomcat-resources/src/main/resources/stop_eXo.sh [hunk 1]
```diff
@@ -66,6 +66,8 @@ while [ "$1" != "" ]; do
     shift
 done
 
-export CATALINA_PID=${CATALINA_PID:-"$PRGDIR/temp/catalina.pid"}
+if [ -x "$PRGDIR/temp/catalina.pid" ]; then
```
Comment (thomasdelhomenie) [inline plf-tomcat-resources/src/main/resources/stop_eXo.sh]: Do catalina.pid really have to be executable ? Shouldn't it be -w ?

File: plf-tomcat-resources/src/main/resources/stop_eXo.sh [hunk 2]
```diff
@@ -66,6 +66,8 @@ while [ "$1" != "" ]; do
...
+  export CATALINA_PID=${CATALINA_PID:-"$PRGDIR/temp/catalina.pid"}
+fi
```
Comment (thomasdelhomenie) [inline plf-tomcat-resources/src/main/resources/stop_eXo.sh]: What is the purpose of this change ?
If the file $PRGDIR/temp/catalina.pid does not exist, the CATALINA_PID variable will keep its current value which can be a custom location where the catalina.pid does not exist also.

**Review by thomasdelhomenie — CHANGES_REQUESTED** [review]

File: plf-tomcat-resources/src/main/resources/stop_eXo.sh (hunk 1 above)
Comment (boubaker) [reply plf-tomcat-resources/src/main/resources/stop_eXo.sh]: -f used instead

File: plf-tomcat-resources/src/main/resources/stop_eXo.sh (hunk 2 above)
Comment (boubaker) [reply plf-tomcat-resources/src/main/resources/stop_eXo.sh]: If the catalina PID doesn't exist, the stop process will exit without calling stop using Tomcat stop port (8005 by default).
The error will be:
```
Using CATALINA_PID:    ./temp/catalina.pid
$CATALINA_PID was set but the specified file does not exist. Is Tomcat running? Stop aborted.

```

Comment (thomasdelhomenie) [reply plf-tomcat-resources/src/main/resources/stop_eXo.sh]: Yes, but this change also prevents the stop to abort even if we set a different location in  CATALINA_PID ?

Comment (boubaker) [reply plf-tomcat-resources/src/main/resources/stop_eXo.sh]: changed to use variable too

**Review by thomasdelhomenie — DISMISSED** [review]
